code/modules/uplink/uplink_items.dm

```diff
@@ -1324,7 +1427,12 @@ datum/uplink_item/stealthy_weapons/taeclowndo_shoes
 			Nanotrasen crew who spot these suits are known to panic."
 	item = /obj/item/clothing/suit/space/hardsuit/syndi
 	cost = 8
-	exclude_modes = list(/datum/game_mode/nuclear) //you can't buy it in nuke, because the elite hardsuit costs the same while being better
+	exclude_modes = list(/datum/game_mode/nuclear)
+
+/datum/uplink_item/suits/hardsuit/nukeop
```

Comment:
Useless they have hardsuits in the ship

Reply:
At highpop, they can be used to supply reinforcements. Not harmful to have em.

Reply:
they get like 7 suits tho. But I get ya point